code/Gamemodes/Bowling/Entities/BowlingBallCarriable.cs

```diff
@@ -1,10 +1,13 @@
 ﻿namespace Sports;
 
-public class BowlingBallCarriable : BaseCarriable
+public partial class BowlingBallCarriable : BaseCarriable
 {
 	public BowlingBall BowlingBall { get; set; }
-
-	TimeSince timeSinceLastThrow = 0;
+	[Net]
+	public bool HasThrown { get; set; }
+	[Net, Predicted]
+	TimeSince TimeSinceLastThrow { get; set; }
+	BowlingPlayer BowlingPlayer => Owner as BowlingPlayer;
```

Comment:
And here too imo

Reply:
this probably should also have accesors.
so private public or anything